### ISSUE TITLE
Refactor feed viewer components

### ISSUE THREAD

**Issue by landonreed** [conveyal/datatools-ui#309]
<a href="https://github.com/evansiroky"><img src="https://avatars1.githubusercontent.com/u/3112493?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [evansiroky](https://github.com/evansiroky)**
_Monday Jun 25, 2018 at 15:20 GMT_
_Originally opened as https://github.com/catalogueglobal/datatools-ui/pull/200_

----

- Refactor route viewer to show trips per hour
- Refactor pattern viewer to show trips per hour
- Refactor stops viewer to work with changes to route and trip viewers
- Add flow typing to reducers

----

_**[evansiroky](https://github.com/evansiroky)** included the following code: https://github.com/catalogueglobal/datatools-ui/pull/200/commits_

**Comment by landonreed** [comment]
<a href="https://github.com/codecov-io"><img src="https://avatars0.githubusercontent.com/u/8655789?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [codecov-io](https://github.com/codecov-io)**
_Monday Jun 25, 2018 at 17:58 GMT_

----

# [Codecov](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200?src=pr&el=h1) Report
> Merging [#200](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200?src=pr&el=desc) into [dev](https://codecov.io/gh/catalogueglobal/datatools-ui/commit/904ccd597c511e78d387b4efbbd4c04a4d784e74?src=pr&el=desc) will **increase** coverage by `3.56%`.
> The diff coverage is `27.43%`.

[![Impacted file tree graph](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/graphs/tree.svg?token=P6y8RDP2dF&src=pr&width=650&height=150)](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200?src=pr&el=tree)

```diff
@@           Coverage Diff            @@
##             dev    #200      +/-   ##
========================================
+ Coverage   0.03%    3.6%   +3.56%     
========================================
  Files        316     325       +9     
  Lines      10935   11304     +369     
  Branches    3846    3952     +106     
========================================
+ Hits           4     407     +403     
+ Misses      8956    8949       -7     
+ Partials    1975    1948      -27
```


| [Impacted Files](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [lib/gtfs/util/graphql.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL2d0ZnMvdXRpbC9ncmFwaHFsLmpz) | `0% <ø> (ø)` | :arrow_up: |
| [lib/editor/util/map.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL2VkaXRvci91dGlsL21hcC5qcw==) | `9.52% <ø> (+9.52%)` | :arrow_up: |
| [lib/gtfs/actions/general.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL2d0ZnMvYWN0aW9ucy9nZW5lcmFsLmpz) | `0% <ø> (ø)` | :arrow_up: |
| [lib/editor/util/timetable.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL2VkaXRvci91dGlsL3RpbWV0YWJsZS5qcw==) | `94.11% <ø> (+94.11%)` | :arrow_up: |
| [lib/gtfs/actions/patterns.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL2d0ZnMvYWN0aW9ucy9wYXR0ZXJucy5qcw==) | `0% <0%> (ø)` | :arrow_up: |
| [...ib/manager/components/reporter/containers/Stops.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL21hbmFnZXIvY29tcG9uZW50cy9yZXBvcnRlci9jb250YWluZXJzL1N0b3BzLmpz) | `0% <0%> (ø)` | :arrow_up: |
| [lib/gtfs/actions/shapes.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL2d0ZnMvYWN0aW9ucy9zaGFwZXMuanM=) | `0% <0%> (ø)` | |
| [lib/admin/reducers/index.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL2FkbWluL3JlZHVjZXJzL2luZGV4Lmpz) | `0% <0%> (ø)` | :arrow_up: |
| [...nager/components/reporter/components/StopLayout.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL21hbmFnZXIvY29tcG9uZW50cy9yZXBvcnRlci9jb21wb25lbnRzL1N0b3BMYXlvdXQuanM=) | `0% <0%> (ø)` | :arrow_up: |
| [lib/gtfs/selectors/index.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree#diff-bGliL2d0ZnMvc2VsZWN0b3JzL2luZGV4Lmpz) | `0% <0%> (ø)` | :arrow_up: |
| ... and [102 more](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200/diff?src=pr&el=tree-more) | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200?src=pr&el=footer). Last update [904ccd5...fa146d8](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/200?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by landonreed** [comment]
<a href="https://github.com/landonreed"><img src="https://avatars2.githubusercontent.com/u/2370911?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [landonreed](https://github.com/landonreed)**
_Monday Jul 02, 2018 at 20:17 GMT_

----

Closing because fixes for this were applied in #202.
